dsb-gui/src/test/java/com/spectralogic/dsbrowser/gui/services/jobService/JobServiceTest.kt

```diff
@@ -89,9 +89,9 @@ class JobServiceTest {
     fun totalJobTest() {
         var total = 100.00
         jobService!!.jobSizeObservable().subscribe(Consumer { t: Number -> total = t.toDouble() })
-        assertThat(total).isEqualTo(PERCENT_INITIAL)
+        assertThat(total).isEqualTo(0.0)
```

Comment:
why is this changed from a const to a magic number?

Reply:
Personally, I think it makes the tests clearer...